lib/screens/explore_screens/regions_screens/edit_region_event/edit_region_event_time_and_date.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:seeds/components/flat_button_long.dart';
+import 'package:seeds/datasource/remote/model/firebase_models/region_event_model.dart';
+import 'package:seeds/domain-shared/event_bus/event_bus.dart';
+import 'package:seeds/domain-shared/event_bus/events.dart';
+import 'package:seeds/domain-shared/page_command.dart';
+import 'package:seeds/domain-shared/ui_constants.dart';
+import 'package:seeds/screens/create_region_event_screens/components/date_time_row.dart';
+import 'package:seeds/screens/explore_screens/regions_screens/edit_region_event/interactor/viewmodel/edit_region_event_bloc.dart';
+
+class EditRegionEventTimeAndDate extends StatelessWidget {
+  const EditRegionEventTimeAndDate({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final event = ModalRoute.of(context)!.settings.arguments as RegionEventModel?;
+
+    return BlocProvider(
+        create: (_) => EditRegionEventBloc(event!),
+        child: BlocConsumer<EditRegionEventBloc, EditRegionEventState>(
+          listenWhen: (_, current) => current.pageCommand != null,
+          listener: (context, state) {
+            final pageCommand = state.pageCommand;
+            if (pageCommand is ShowErrorMessage) {
+              eventBus.fire(ShowSnackBar(pageCommand.message));
+            }
+            BlocProvider.of<EditRegionEventBloc>(context).add(const ClearEditRegionEventPageCommand());
+          },
+          builder: (context, state) {
+            return Scaffold(
```

Comment:
move the Scaffold outside of the bloc consumer
AppBar(title: const Text("Edit Event")), is static and doesnt need to change

Reply:
done